app/assets/javascripts/backlink.js

```diff
@@ -0,0 +1,30 @@
+// prevent resubmit warning
+// replace history with corrected url
+
+
+var docReferrer = document.referrer;
+
+if (
+    window.history &&
+    window.history.replaceState &&
+    typeof window.history.replaceState === 'function'
+) {
+
+    var urlSegment =    window.location.href.substring(window.location.href.lastIndexOf('/') + 1);
+
+    switch(urlSegment) {
```

Comment:
These are edge cases for specific journeys?

Reply:
yep that's right